CONTRIBUTING.md

```diff
@@ -1,34 +1,70 @@
 
-## Taking Issues
+The following are all good examples of pull request titles:
 
-It is highly recommended to request an assignment before beginning work on any issues to avoid conflicts with other community members.
+```text
+feat(services/gcs): Add start-after support for list
+docs: add hdfs classpath related troubleshoot
+ci: Mark job as skipped if owner is not apache
+fix(services/s3): Ignore prefix if it's empty
+refactor: Polish the implementation of webhdfs
+```
 
-It is encouraged to post your work progress in the issues comment so that the community can stay informed about ongoing developments and offer any necessary assistance.
+#### Reviews & Approvals
 
-It is possible that your assignment has been removed after 14 days of no updates. Please note that this is not a punishment or dissatisfaction from the community. We understand that life can be challenging, and things may not always go as planned.
+All pull requests should be reviewed by at least one OpenDAL committers.
 
-By removing the assignment, we create space for other community members to take over. Additionally, we do not frequently check the assignment due date, so please communicate any stale issues through the issue tracker.
+#### Merge Style
 
-## Sending PRs
+All pull requests are squashed and merged. We generally discourage large pull requests that are over 300-500 lines of diff. If you would like to propose a change that is larger we suggest coming onto our [Discussions](https://github.com/apache/incubator-opendal/discussions) and discuss it with us. This way we can talk through the solution and discuss if a change that large is even needed! This will produce a quicker response to the change and likely produce code that aligns better with our process.
 
-If you find an issue you would like to fix, open a pull request. For code changes, please refer to the particular component's development guide.
+### CI
 
-We will try our best to review your PRs within 24 hours, and often within 8 hours.
+Currently, Vector uses GitHub Actions to run tests. The workflows are defined in `.github/workflows`.
+
+## Next steps
+
+Visiting **[DEVELOPING.md](DEVELOPING.md)** to prepare Everything necessary to develop.
+
+## Code of Conduct
+
+We expect all community members to follow our [Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
 
-If a PR does not receive a response within 24 hours, it will be converted to draft status. To indicate that you are ready for review, please mark the PR as `ready for review`.
+---
 
+*This document is highly inspired by [Vector](https://github.com/vectordotdev/vector/)*
```

Comment:
It's great for the credit. But I think it's also really confused. Maybe better to remove this line.

Reply:
DONE.